cmd/dep/godep_importer_test.go

```diff
@@ -395,3 +338,13 @@ func equalImports(a, b []godepPackage) bool {
 
 	return true
 }
+
+// projectRootPtr takes a ProjectRoot value and returns a pointer.
+func projectRootPtr(p gps.ProjectRoot) *gps.ProjectRoot {
```

Comment:
Here's how you can deal with these string types (gps.Revision and gps.ProjectRoot) without introducing these helper functions:

1. Change the want/got in the testcase matrix from pointers to the struct type, e.g. `*gps.Revision` to `gps.Revision`.
1. Update the assignments to not use `proejctRootPtr`/`revisionPtr`.
1. Update the asserts to compare the want testcase value against the empty string, instead of nil, e.g. 
    ```
    if testCase.notWantedProjectRoot != "" {
    ```